_config/config.yml

```diff
@@ -7,7 +7,9 @@ After:
 Director:
   rules:
    'timednotice//$Action/$ID/$OtherID': 'TimedNoticeController'
-
+Page:
+  extensions:
+    - TimedNoticedPageExtension
```

Comment:
Good point, no need for this extension to be applied unless the developer wants to make use of the website/frontend notices